### ISSUE TITLE
Consider a better name

### ISSUE THREAD

**Issue by FlyinPancake** [zhangyuannie/butter#10]
### Describe the problem your feature would solve
There is a project called `butter` that is a btrfs subvolume manager. The clashing names can be confusing for some.

### Describe the solution you'd like
A discussion about possible names for this project :smile:

**Comment by allaeddineomc** [comment]
what about the name 'snapshots'

**Comment by TimLovesTech** [comment]
Or btrsnapshots? It's not taken yet, and makes it clear it's a btrfs project and targeting snapshots.

**Comment by zhangyuannie** [comment]
While not perfect, Butter is kinda okay as the other `butter` is written in shell scripts while Butter is a GUI application. The other `butter` also hasn't been updated for quite some time and is not packaged on most distros that I'm aware of.

Butter also owns some directories such as `/etc/butter` to store schedule rules, so changing the name can be complicated.

All that being said, I'm not totally against a new name for better discoverability, but it should be worth it. It should at least:
- be composed of a single word
- follow [GNOME HIG](https://developer.gnome.org/hig/guidelines/app-naming.html)
- I hope for this app to become a superset of the `btrfs` utilities, not scoped to just snapshots.

My smol brain hasn't found a better name back then. 😝

**Comment by andyholmes** [comment]
You could go for something like "margarine" or "shortening", or an object like "churner"?

**Comment by Scotty-Trees** [comment]
"Cadience" (not a real word, but a play on the root word)

(not to be confused with Cadence, which happens to be some [Uber service](https://github.com/uber/cadence))

It's not butter or milk related, but with the recent [Gradience](https://github.com/GradienceTeam/Gradience) libadwaita tool, I think it has a nice similarity, but different touch to it and Cadence isn't named after anything as far as I can tell. Just my 2 cents. I'm sure there's far better names, but honestly I still happen to like Butter.

**Comment by zhangyuannie** [comment]
> "Cadience" (not a real word, but a play on the root word)

Yeah, searching Cadience on Google yield Cadence directly. The squiggly red line from auto correct doesn't help either. Ideally, it should be a real noun per HIG.

> It's not butter or milk related

Doesn't have to honestly. I guess it just has to relate to subvolume/partition/disk management in general.

> I still happen to like Butter.

Same, feel like it *flows* better.

**Comment by allaeddineomc** [comment]
what about "cow" as in copy on write ?

**Comment by bluej121999** [comment]
What about btrsnaps?

**Comment by iitzrohan** [comment]
How about btrshots?

**Comment by deusnovus** [comment]
The name, icon and vibe fits GNOME perfectly, don't change a thing!
